docs/api.md

```diff
@@ -124,6 +124,7 @@
       - [bot.scoreboard](#botscoreboard)
       - [bot.controlState](#botcontrolstate)
     - [Events](#events)
+      - ["particle" (particle)](#particle-particle)
```

Comment:
it says not to edit the toc at the top, we have a tool hat makes the toc

Reply:
I will uncommit it